Automattic/wp-calypso
# Update concierge session upsell messaging to accommodate the ecommerce plan

#### Changes proposed in this Pull Request

* Update concierge session upsell messaging to accommodate the ecommerce plan

#### Testing instructions

* Go to a concierge upsell screen (`https://wordpress.com/me/concierge/<SITE URL>/book`) as a free/blogger/personal/premium user and confirm the upsell message now says "Only sites on a Business or higher plan are eligible for a Concierge session"
